src/products/components/ProductVariantNavigation/ProductVariantNavigation.tsx

```diff
@@ -46,91 +44,125 @@ const ProductVariantNavigation: React.FC<
+                  display="flex"
+                  alignItems="center"
+                  gap={5}
                 >
-                  <FormattedMessage {...messages.addVariant} />
-                </Button>
-              </TableCell>
-            </TableRowLink>
-          ) : (
-            <TableRowLink>
-              <TableCellAvatar
-                alignRight
-                className={clsx(
-                  classes.colAvatar,
-                  classes.rowActive,
-                  classes.noHandle,
-                  {
-                    [classes.firstVariant]: variants?.length === 0,
-                  },
-                )}
-                thumbnail={null}
-                colSpan={2}
-              />
-              <TableCell className={classes.colName}>
-                <FormattedMessage {...messages.newVariant} />
-              </TableCell>
-            </TableRowLink>
-          )}
-        </SortableTableBody>
-      </ResponsiveTable>
-    </Card>
+                  <GripIcon />
+                  {thumbnail?.url ? (
+                    <Box
+                      as="img"
+                      width={10}
+                      height={10}
+                      objectFit="cover"
+                      borderRadius={2}
+                      borderColor="neutralHighlight"
+                      borderStyle="solid"
+                      borderWidth={1}
+                      padding={1}
+                      flexShrink="0"
+                      src={thumbnail.url}
+                    />
+                  ) : (
+                    <ImagePlaceholder />
+                  )}
+                  <Box>
+                    <Text wordBreak="break-word">
+                      {variant ? variant.name || variant.sku : <Skeleton />}
+                    </Text>
+                    {isDefault && (
+                      <Text
+                        display="block"
+                        variant="caption"
+                        color="textNeutralSubdued"
+                      >
+                        {intl.formatMessage(messages.defaultVariant)}
+                      </Text>
+                    )}
+                  </Box>
+                </Box>
+                <Divider />
+              </SortableElement>
+            </Link>
+          );
+        })}
+        {!isCreate ? (
+          <Button
+            marginTop={4}
+            marginLeft={6}
+            variant="secondary"
+            onClick={() => navigate(productVariantAddUrl(productId))}
+            data-test-id="button-add-variant"
+          >
+            <FormattedMessage {...messages.addVariant} />
+          </Button>
+        ) : (
+          <Box
+            paddingX={5}
+            paddingY={3}
+            display="flex"
+            alignItems="center"
+            gap={5}
+            borderLeftStyle="solid"
+            __borderLeftWidth={2}
+            __borderColor={vars.colors.background.highlightDim}
```

Comment:
**question**: why is background token used here for border?

Reply:
Because this color doesn't exists in border colors so I cannot select it